lib/inventory_refresh/save_collection/saver/retention_helper.rb

```diff
@@ -57,7 +57,7 @@ def ids_array(records)
       #
       # @param records [Array<Hash>] Records we want to archive.
       def archive_all_records!(records)
-        inventory_collection.model_class.where(:id => records.map { |x| x[:id] }).update_all(:deleted_on => Time.now.utc)
+        inventory_collection.model_class.where(:id => records.map { |x| x[:id] }).update_all(:archived_on => Time.now.utc)
```

Comment:
Wonder if we should get the archived_on column name from the IC in case someone needs to change it?

Reply:
We could probably do that later.